packages/amazonq/package.json

```diff
@@ -614,6 +607,45 @@
                 "command": "aws.amazonq.showHistoryInHub",
                 "title": "Show Job Status",
                 "icon": "$(history)"
+            },
+            {
+                "command": "aws.amazonq.selectCustomization",
+                "title": "%AWS.codewhisperer.customization.notification.new_customizations.select%",
+                "category": "%AWS.amazonq.title%"
+            },
+            {
+                "command": "aws.amazonq.gettingStarted",
```

Comment:
why does this need to be the command palette? vscode already exposes all walkthroughs via `Welcome: Open Walkthrough...`